docs/sphinx-docs/build_sphinx.py

```diff
@@ -319,6 +324,7 @@ def retrieve_user_docs():
                 fromhere=os.path.join(SASMODELS_DEST_IMG,files)
                 tohere=os.path.join(SASMODELS_DEST_BUILDIMG,files)
                 shutil.copy(fromhere,tohere)
+    else: print "no source directory",SASMODELS_SOURCE_MODELS," was found.\n !!!!NO MODEL DOCS WILL BE BUILT!!!!"
```

Comment:
print should be a function. spaces after both commas needed. line needs to be <80 chars. print on new line